lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -10,17 +10,20 @@ package Yam::Agama::Pom::GrubMenuAgamaPage;
 use strict;
 use warnings;
 use testapi;
+use utils 'is_uefi_boot';
 
 sub new {
     my ($class, $args) = @_;
     return bless {
         grub_menu_base => $args->{grub_menu_base},
         tag_agama_installer_highlighted => 'grub-menu-agama-installer-highlighted',
+        tag_agama_grub_menu_suse => 'grub-menu-agama-suse-brand',
     }, $class;
 }
 
 sub expect_is_shown {
     my ($self) = @_;
+    assert_screen($self->{tag_agama_grub_menu_suse}, 60) if is_uefi_boot;
```

Comment:
Do we need the `if is_uefi_boot`? We're checking for the SUSE logo, that should also appear on non uefi boot. And then we could also remove the `unless...` in the next line because that was introduced to prevent sending key down events as long as we don't see a boot menu.

Reply:
It is ok to remove the condition for is_uefi_boot, all assert the SUSE logo firstly. 
For the next line unless it is needed since send_key_until_needlematch will first send key then match the needle, if without it will cause the currently test in production group failed.

Reply:
As I wrote in the ticket `send_key_until_needlematch` is a misleading name because the code first checks the needle and then, if no match it sends the key. See the code here: https://github.com/os-autoinst/os-autoinst/blob/4c80baea32ba84f8b5c741f867e2caa277ec645c/testapi.pm#L1340

Reply:
Ok, updated code. Now only add a needle and do not need check_screen. Updated VRs also.